src/gui/tray/UnifiedSearchResultItemSkeleton.qml

```diff
@@ -14,8 +14,8 @@ RowLayout {
     property int titleFontSize: Style.topLinePixelSize
     property int sublineFontSize: Style.subLinePixelSize
 
-    property string titleColor: "black"
-    property string sublineColor: "grey"
+    property string titleColor: Style.ncTextColor
+    property string sublineColor: Style.ncSecondaryTextColor
```

Comment:
can you check if those properties could be made of type `color`

Reply:
Changed to color